atd-vze/.env

```diff
@@ -3,5 +3,5 @@ REACT_APP_HASURA_ENDPOINT="http://localhost:8084/v1/graphql"
 REACT_APP_AUTH0_DOMAIN=atd-datatech.auth0.com
 REACT_APP_AUTH0_CLIENT_ID=2qbqz2sf8L9idBOwn0d5YA9efNgQbL7c
 REACT_APP_MAPBOX_TOKEN="pk.eyJ1Ijoiam9obmNsYXJ5IiwiYSI6ImNqemU4OGQzaTBlZjczYm1rZWg1c2lidTcifQ.2eooq4Q7dv_1Le5la9D96Q"
-REACT_APP_NEARMAP_KEY="YzI5YTM1ZWMtMWRkNy00NGU4LTkxMWQtNDI3ZDcxNjVjMzg3"
+REACT_APP_NEARMAP_KEY="NGQ4NGUwOWQtZTlhZS00Y2FkLTkyMWItOGIxYWJlZDYyNTcy"
```

Comment:
The local environment had an older key that is probably decommissioned. I tested this key, and it also does not work locally.